deploy/ocm-agent-operator-managednotifications/10-managednotifications-cr.yaml

```
@@ -12,4 +12,28 @@ spec:
       resolvedBody: |-
         Your cluster's ElasticSearch deployment is detected as being at safe disk consumption levels and no additional action on this issue is required.
       severity: Info
-      summary: "ElasticSearch reaching disk capacity" 
\ No newline at end of file
+      summary: "ElasticSearch reaching disk capacity" 
+    - activeBody: |-
+        Your cluster requires you to take action as its ElasticSearch cluster has hit the high watermark due to low free disk space. Some shards will be re-allocated to different nodes if possible.  Please consult the following documentation for reference: https://docs.openshift.com/container-platform/latest/logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.html#Elasticsearch-Node-Disk-High-Watermark-Reached
+      name: ElasticsearchNodeDiskWatermarkReached
+      resendWait: 24
+      resolvedBody: |-
+        Your cluster has no more ElasticsearchNodeDiskWatermarkReached alert and no additional action on this issue is required.
```

Removing references to alerts.
```suggestion
        Your cluster's ElasticSearch cluster is no longer breaching its node disk high watermark. No additional action on this issue is required.
```